src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -52,7 +52,7 @@ def load_arguments(self, _):
         c.argument('https_proxy', options_list=['--proxy-https'], arg_group='Proxy', help='Https proxy URL to be used.')
         c.argument('http_proxy', options_list=['--proxy-http'], arg_group='Proxy', help='Http proxy URL to be used.')
         c.argument('no_proxy', options_list=['--proxy-skip-range'], arg_group='Proxy', help='List of URLs/CIDRs for which proxy should not to be used.')
-        c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well)')
+        c.argument('proxy_cert', options_list=['--proxy-cert', '--custom-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the any additional certificate file (for proxy as well)')
```

Comment:
--custom-ca-cert ?

Reply:
'Path to the any additional certificate file (for proxy as well)', the help text here could be same as the previous one -> 'Path to the certificate file for proxy or custom CA certificate'